### DISCUSSION TITLE
Previous Month Time Intelligence not referencing properly

### DISCUSSION THREAD

**Discussion by robert-bugna** [sql-bi/Bravo#661] [Q&A]
I'm using the latest version of Bravo which I noticed the previous month (April) values is not reflected properly in PM for May 2023.
Could you point out if I missed anything on my end?

![image](https://github.com/sql-bi/Bravo/assets/104728199/76703e0f-46ad-43f3-be06-36e7b05d9950)

**Comment by albertospelta** [comment]
@robert-bugna What calendar template did you use to create the date table ? One of the predefined templates or a customized one?
Could you share a minimal .pbix file with some dummy data that reproduces this behavior?
